packages/suite/src/utils/suite/solanaStaking.ts

```diff
@@ -0,0 +1,79 @@
+// @ts-expect-error: Could not find a declaration file for module '@everstake/wallet-sdk/solana'.
+import { stake } from '@everstake/wallet-sdk/solana';
```

Comment:
We might want to have this method (and other that are used) in our codebase somewhere, since it composes transactions cc @tomasklim